src/main/java/at/ac/tuwien/kr/alpha/common/predicates/Evaluable.java

```diff
@@ -0,0 +1,11 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+import at.ac.tuwien.kr.alpha.grounder.Substitution;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface Evaluable {
+	boolean evaluate(List<Term> terms, Substitution substitution);
```

Comment:
Similar as External, the Evaluable should be able to return variable bindings. According to the standard (but not yet realized in Alpha), `=` may bind a single variable on either side to the value of the other side.